### DISCUSSION TITLE
Hide zsh-autosuggestions results when entering menu

### DISCUSSION THREAD

**Discussion by tshu-w** [marlonrichert/zsh-autocomplete#316] [Ideas]
<img width="1591" alt="Screen Shot 2021-07-31 at 13 25 55" src="https://user-images.githubusercontent.com/13161779/127729754-b726b557-7597-4b16-8f76-86faf02753bb.png">

**Comment by marlonrichert** [comment]
They are hidden for me when entering the completion menu. 🤔 

Could you open a bug report for this, with a reproducible test case?

**Reply by tshu-w** [reply]
@marlonrichert OK, I'll try to reproduce it with minimal configuration

**Reply by marlonrichert** [reply]
#317
